glambie/processing/process_global_results.py

```diff
@@ -86,6 +85,14 @@ def run_global_results(glambie_run_config: GlambieRunConfig,
             region=REGIONS["global"], data_group=GLAMBIE_DATA_GROUPS["consensus"], csv_file_name="global_gt.csv")
         global_timeseries_gt.save_data_as_csv(csv_output_file)
 
+        # save out regional results homogenized to calendar year (in mwe and in gt)
```

Comment:
Very minor. Comment on line 57 should reflect plot and save to csv

Reply:
done